worlds/landstalker/Options.py

```diff
@@ -0,0 +1,227 @@
+from typing import Dict
+
+from Options import Choice, Range, Option, Toggle, DeathLink, DefaultOnToggle
+
+
+class LandstalkerGoal(Choice):
+    """
+    The goal to accomplish in order to complete the seed.
+    - Beat Gola: beat the usual final boss (same as vanilla)
+    - Reach Kazalt: find the jewels and take the teleporter to Kazalt
+    - Beat Dark Nole: the door to King Nole's fight brings you into a final dungeon with an absurdly hard boss you have to beat to win the game
+    """
+    display_name = "Goal"
+
+    option_beat_gola = 0
+    option_reach_kazalt = 1
+    option_beat_dark_nole = 2
+
+    default = 0
+
+
+class JewelCount(Range):
+    """
+    Determines the number of jewels to find in order to be able to reach Kazalt
+    """
+    display_name = "Jewel count"
+    range_start = 0
```

Comment:
Yes, it's a perfectly reasonable choice in case we'd want to just skip that requirement and be able to go to the final dungeon without having to collect the jewels.